### PR TITLE
don't submit selectItem values (fixes issue #192)

### PR DESCRIPTION
**Pull request by vfigueiro**
This fixes issue #192

### CODE REVIEW

**Comment by prog666** [comment]
contribute here https://github.com/prog666/multiple-select

File: jquery.multiple.select.js [hunk 1]
```diff
@@ -261,6 +261,16 @@
                 if (that.options.single && that.options.isOpen && !that.options.keepOpen) {
                     that.close();
                 }
+
+                if (that.options.single) {
```
Comment (wenzhixin) [inline jquery.multiple.select.js]: What problem are these codes fix?

**Comment by wenzhixin** [comment]
Update to `data-name=` in other's PR, you can rebase your PR to update.

**Comment by wenzhixin** [comment]
Thanks! Update to the latest version.

**Pull request closed without merging**